wgsl/syntax/builtin_value_name_token.syntax.bs.include

```diff
@@ -0,0 +1,5 @@
+<div class='syntax' noexport='true'>
+  <dfn for=syntax>builtin_value_name_token</dfn> :
```

Comment:
I prefer not using token in the rule name similar to diagnostic parameters.

Reply:
Thank you very much, it is more concise without `_token`, so I did incorporate with new commit!